packages/labs/analyzer/test-files/ts/properties/src/element-a.ts

```diff
@@ -99,6 +99,11 @@ export class ElementA extends LitElement {
   @property()
   union: LocalClass | HTMLElement | ImportedClass;
 
+  @state()
+  private _stateField: number;
+
+  #privateField = 0;
```

Comment:
Is this enough to make the test fail without the patch (seems like there is no test code in this PR that explicitly checks it)

Reply:
Yes, just the presence of the field causes a diagnostic before this change.